spec/lib/requirements/topic_checker_spec.rb

```diff
@@ -5,18 +5,20 @@
 
   describe "#pre_publish_issues" do
     it "returns no issues if there are none" do
-      edition = build :edition, :publishable
-      issues = Requirements::TopicChecker.new(edition.document).pre_publish_issues
+      document = create(:document, :with_current_edition)
+      issues = Requirements::TopicChecker.new(document).pre_publish_issues
```

Comment:
I'm curious, why did this test need to change?

Reply:
It's because the document, which we need to pass to TopicChecker didn't have an edition associated with it and we were getting the following error:
```
Failure/Error: if document.editions.last.document_type.topics && document.topics.none?    
     NoMethodError:
       undefined method `document_type' for nil:NilClass
```

Reply:
Would changing `build :edition` to `create` work? I think the edition factory creates a document if one doesn't exist, and it would match the set-up in the other tests.

Reply:
this does work. Have adjusted :)